addons/hr_timesheet/report/project_report.py

```diff
@@ -25,3 +25,10 @@ def _group_by(self):
             t.effective_hours,
             planned_hours
             """
+
+    @api.model
+    def _fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu=False):
+        result = super(ReportProjectTaskUser, self)._fields_view_get(view_id=view_id, view_type=view_type, toolbar=toolbar, submenu=submenu)
+        if view_type == 'pivot' and self.env.company.timesheet_encode_uom_id == self.env.ref('uom.product_uom_day'):
+            result['arch'] = self.env['project.task']._apply_time_label(result['arch'], related_model='report.project.task.user')
```

Comment:
You can also use `self._name` to have the name of the model 🙂 
```suggestion
            result['arch'] = self.env['project.task']._apply_time_label(result['arch'], related_model=self._name)
```